executor/hash_table.go

```diff
@@ -37,11 +38,14 @@ import (
 // hashContext keeps the needed hash context of a db table in hash join.
 type hashContext struct {
 	// allTypes one-to-one correspondence with keyColIdx
-	allTypes  []*types.FieldType
-	keyColIdx []int
-	buf       []byte
-	hashVals  []hash.Hash64
-	hasNull   []bool
+	allTypes        []*types.FieldType
+	keyColIdx       []int
+	naKeyColIdx     []int
+	buf             []byte
+	hashVals        []hash.Hash64
+	hasNull         []bool
+	naHasNull       []bool
+	naColNullBitMap []*bitmap.ConcurrentBitmap
```

Comment:
why need a concurrentBitmap here?

Reply:
yes, it's unnecessary here

Reply:
If it's unnecessary, I think we should not use concurrentBitMap here, please use a normal bitmap instead.